apps/src/applab/applab.js

```diff
@@ -849,12 +850,9 @@ Applab.init = function (config) {
 
   // TODO - move into AppLabView and put necessary props into store
   var codeWorkspace = (
-    <CodeWorkspace
-      localeDirection={studioApp.localeDirection()}
-      editCode={!!config.level.editCode}
-      readonlyWorkspace={Applab.reduxStore.getState().level.isReadOnlyWorkspace}
-      showDebugger={showDebugButtons || showDebugConsole}
-    />
+    <Provider store={Applab.reduxStore}>
+      <ConnectedCodeWorkspace/>
+    </Provider>
```

Comment:
Interesting.  Presumably you don't need the additional `Provider` here since this component is eventually rendered in the `AppLabView` context which is already wrapped.  I guess it wouldn't hurt though.  Did you find this was necessary?